traffic_ops/traffic_ops_golang/cachegroup/cachegroups.go

```diff
@@ -444,7 +445,7 @@ func (cg *TOCacheGroup) Read(h http.Header, useIMS bool) ([]interface{}, error,
 	}
 
 	if useIMS {
-		runSecond, maxTime := ims.TryIfModifiedSinceQuery(cg.APIInfo().Tx, h, queryValues, selectMaxLastUpdatedQuery(where))
+		runSecond, maxTime = ims.TryIfModifiedSinceQuery(cg.APIInfo().Tx, h, queryValues, selectMaxLastUpdatedQuery(where))
```

Comment:
ok, just want to make sure that if you don't provide the following header:

```
-H "If-Modified-Since: [date]"
```

then you get back 

```
< last-modified: Mon, 01 Jan 0001 00:00:00 UTC
```

Reply:
just want to make sure that's the expected result.

Reply:
Correct, I set the "last-modified" in the response only if we get a valid IMS request header, and if the "use_ims" flag is set to true in the conf file.

Reply:
i guess i expected that the `last-modified` key would not even be in the response at all if IMS was not in the request header

Reply:
Well, I have the logic set currently so that if we get a valid (not nil) pointer in the `maxtime` variable, we set the `last-modified` value to that. So, if we get back an empty maxTime (which is the case if you dont supply an IMS header, or if the flag is set to false), we set the last-modified to `Mon, 01 Jan 0001 00:00:00 UTC`.